### DISCUSSION TITLE
@livewireStyles and @livewireScripts tags are not processed

### DISCUSSION THREAD

**Discussion by danielzzz** [livewire/livewire#4429] [1. Help] [answered]
Hi, I have laravel 8.54
and "livewire/livewire": "^2.8",

but it seems that the tags to include livewire are not being processed. the rendered page just show the directives.

I have checked the forums for solutions but these did not work for me:
https://forum.laravel-livewire.com/t/resolved-livewirescripts-and-livewirestyles-showing-on-page/147/11
https://forum.laravel-livewire.com/t/working-on-local-but-not-on-staging/1201/5
.. and others

things I tried,
- composer dump-autoload (it discovers livewire, and livewire artisan commands do work)
- php artisan cache:clear
- removing manually bootstrap/cache files
- calling the directives with and without ()
- even tried to include the scripts and styles manually, but it's pointless as the component directives are not processed either.

it seems that registerBladeDirectives() from LivewireServiceProvider is being called, but then somehow it's ignored in the layout.
public static function livewireStyles($expression) from LivewireBladeDirectives.php is never called

calling the functions directly does work:

{!! \Livewire\Livewire::scripts() !!}
so I guess, the problem must be somewhere with the directive processing in laravel.

I have not figured out how I could load the components this way...

I am not sure what else I could do to debug it...

please advice, thank you!
dan

**Comment by joshhanley** [comment]
@danielzzz hmm that seems strange!

Any chance you could create a small example repo which reproduces the issue with as little code as possible so we can investigate it and test for you to see if it works on our end?

Also what environment are you running in? Local or on a server? Windows/mac/linux? Valet/ homestead/ etc?

Thanks!

**Comment by danielzzz** [comment, marked as answer]
hi, I have found the problem. 

There was a conflict with another library which caused laravel to ignore there directives. 
please check this issue for details:
https://github.com/TerrePorter/StringBladeCompiler/issues/73#issuecomment-999674373

best, dan
